eclass/qt5-build.eclass

```diff
@@ -231,6 +231,11 @@ qt5-build_src_configure() {
 		qt5_tools_configure
 	fi
 
+	# Workaround for bug #915203
```

Comment:
please also cite QTBUG-111514 for tracking upstream's integration into 5.15 branch.